index.html

```diff
@@ -226,7 +226,7 @@ <h3>About Me</h3>
           />
         </div>
         <div>
-          <input type="email" id="email" placeholder="Email" name="Email" />
+          <input type="email" id="email" placeholder="Email" name="Email address" required />
```

Comment:
It seems like you changed the wrong attribute, please make sure to change `Placeholder` and not `name`!